client.go

```diff
@@ -0,0 +1,179 @@
+package luasandbox
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/calyptia/api/types"
+)
+
+type Client struct {
+	url        string
+	httpClient http.Client
+	nextId     int
+}
+
+type params struct {
+	Events []types.FluentBitLogAttrs `json:"events"`
```

Comment:
```suggestion
	Records []types.FluentBitLogAttrs `json:"records"`
```

I would rename this to `records` to follow fluent-bit docs https://docs.fluentbit.io/manual/pipeline/filters/lua

Reply:
I'd rather not change this now for the same reason [explained above(https://github.com/calyptia/go-lua-sandbox-client/pull/1#discussion_r1036344749).

I will have to modify the server to make use of timestamps (and possibly tags too), so let's do these aesthetic changes in the next iteration

Also note that this is not part of the public API of the client library. The client still accepts FluentBitLog slice, so I will only have to modify the library implementation after we make use of timestamps

Reply:
Not entirely true. The client passes a Lua code and records that have timestamps. In the Lua code we have access to the timestamp:

```lua
function processing_rule(tag, timestamp, record) {
  -- Could do something with the timestamp
}
```

Reply:
Not sure I understand, but yes the lua code has access to tag/timestamp, but it is not currently passed by the server simply because it treats input as a list of records. I have detailed the next changes to support timestamp/tags [here](https://github.com/calyptia/cloud-lua-sandbox/issues/11)

What I meant is that we should do naming changes (which require changing both this client and the server) in the next iteration, which will break the cloud-lua-sandbox API

Reply:
Better to change those names now than later.